sdk/eventhub/Azure.Messaging.EventHubs/src/EventHubProducerClient.cs

```diff
@@ -442,7 +442,7 @@ public virtual Task<string[]> GetPartitionIdsAsync(CancellationToken cancellatio
         ///   Sends a set of events to the associated Event Hub using a batched approach.
         /// </summary>
         ///
-        /// <param name="eventBatch">The set of event data to send.</param>
+        /// <param name="eventBatch">The set of event data to send. You can create a batch by calling CreateBatchAsync.</param>
```

Comment:
I like the idea, but I'd like to suggest a minor tweak to better align with the tone that we've been using in the doc comments.  Maybe something like: `... A batch may be created using <see cref="CreateBatchAsync" />.`

Reply:
SGTM!